### PR TITLE
maintainers: Update Confidential Containers maintainers list & governance members URL

### PR DESCRIPTION
**Pull request by fidencio**
---

 maintainers: Update Confidential Containers maintainers

 Three new members have joined the Confidential Containers Steering
 Committee (Ananya Garg and Vicent Batts, from Microsoft; Zvonko Kaiser,
 from NVIDIA), and Intel has updated its representation (with Fabiano
 Fidêncio replacing Dan Middleton).

---

project-maintainers: Update Confidential Containers URL

We've merged two repos into one, and now the list of the maintainers is
part of the governance file, which is now updated.

---

### CODE REVIEW

**Review by fitzthum — APPROVED** [review]

**Comment by fidencio** [comment]
/hold

As Dan Middleton has been "in the process of resolving fossa access through CNCF help desk", we better wait till that one is fixed before updating the list of maintainers, just in case CNCF drops his access to the help desk.

**Comment by amye** [comment]
That won't effect this, we'll be able to do both.

**Review by magowan — APPROVED** [review]

**Review by dcmiddle — APPROVED** [review]

**Pull request merged**
